src/main/java/sirius/kernel/commons/Files.java

```diff
@@ -146,6 +159,34 @@ public static String getFilenameWithoutExtension(@Nullable String path) {
         return Strings.splitAtLast(getFilenameAndExtension(path), ".").getFirst();
     }
 
+    /**
+     * Determines if the given path is hidden.
+     * <p>
+     * A path is considered hidden if it starts with a dot.
+     *
+     * @param path the path to check
+     * @return <tt>true</tt> if the path is hidden, <tt>false</tt> otherwise
+     */
+    public static boolean isHidden(@Nullable String path) {
```

Comment:
**Suggestion**: rename to `isConsideredHidden`, since we are validating a string and not using any file-system feature.